one-ui/src/main/java/com/marklogic/hub/oneui/controllers/ModelController.java

```diff
@@ -60,4 +62,18 @@ ModelManager modelManager() {
         JsonNode json = modelManager.getModel(modelName);
         return new ResponseEntity<>(json, HttpStatus.OK);
     }
+
+    @RequestMapping(value = "/job-info", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<?> getLatestJobInfoForAllModels() {
```

Comment:
I agree with including this here, as this is job stuff that's specific to models, as opposed to job stuff that is global to a project.